product_categ_image/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2018 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    'name': 'Product Category Image',
+    'version': '10.0.1.0.0',
+    'category': 'Sales',
+    'license': 'AGPL-3',
+    'summary': 'Add image on product category',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```

Comment:
Please change it to `https://github.com/OCA/product-attribute`

Reply:
Done